src/meinberlin/meinberlin/static/stylesheets/scss/components/_error_pages_theme.scss

```diff
@@ -0,0 +1,11 @@
+.error-page-404 {
+    .error-page-image {
+        max-width: 300px;
```

Comment:
I would switch this to `width: 300px` so that the `max-width: 100%` is not overwritten.